docs/GettingStarted.md

```diff
@@ -0,0 +1,103 @@
+# Getting Started
+
+## Installing React Native and its dependencies
+
+This is a targeted summary of setup React Native setup for specifically React Native Windows. See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for full details.
```

Comment:
We should note that these are all PowerShell commands. for Windows Subsystem for Linux (WSL) bash shell, following the Linux instructions in the React Native getting started should work but we may want to say that scenario is not officially supported right now. (I think Fall Creators Update WSL should work great, but we need to test and document it properly.)

Reply:
Good idea, will change

Reply:
This works fine on Command Prompt, nothing specific to Powershell. Would be fine to note that the WSL scenario isn't officially supported.

---
In reply to: [146349032](https://github.com/Microsoft/react-native-windows/pull/1464#discussion_r146349032) [](ancestors = 146349032)